onnxruntime/hosting/http/http_server.cc

```diff
@@ -29,42 +25,44 @@ namespace hosting {
 using handler_fn = std::function<void(std::string, std::string, std::string, HttpContext&)>;
 
 App::App() {
```

Comment:
If you do not pass in the `env`, you have no way to print log in this class. Is it what you expected? Why?

Reply:
Yes. If you look at the first commit for this PR you'll see that I did that at first BUT I had build errors + I don't want that tight of coupling.

The only place we are logging is at startup and when there's errors. We'll create callbacks and pass `env` for that purpose.